coralproject/talk
# [CORL-382] Allow moderators to ban a commenter from the comment stream

## What does this PR do?

Adds a ban user drop down option in the comment stream's moderation drop down.

## How do I test this PR?

- Login as a moderate capable user
- Open the comment stream
- Find a comment with a user you want to ban
- Click the moderation drop down via the top right caret on the comment
- Click the new Ban User option
- Confirm you want to ban them
- User will be banned
- Click on caret again
- Drop down should have replaced the Ban User option with Banned
